scipy/integrate/_quadrature.py

```diff
@@ -24,9 +26,20 @@ def _copy_func(f):
     return g
 
 
-trapz = _copy_func(trapz)
-if trapz.__doc__:
-    trapz.__doc__ = trapz.__doc__.replace('sum, cumsum', 'numpy.cumsum')
+trapezoid = _copy_func(trapezoid)
+if trapezoid.__doc__:
+    trapezoid.__doc__ = trapezoid.__doc__.replace('sum, cumsum', 'numpy.cumsum')
```

Comment:
> Yes, I chose to ignore that - breaking the lines for 1 or 2 characters doesn't make sense here.

My preference is to either conform or add `# noqa: E501`s where necessary, rather than have violations, as it makes writing PEP8-compatible code easier for the next person (and makes reviewing easier because CIs can be green). For the remaining 1-2 char ones in this PR, it seems like it would be pretty easy and not hurt readability much to fix them, e.g.:
```suggestion
    trapezoid.__doc__ = trapezoid.__doc__.replace(
        'sum, cumsum', 'numpy.cumsum')
```

But doing `# noqa: E501` at the end of this line is fine, too.

Ultimately if you think this is overkill or are adamantly opposed to it I can live with selectively ignoring (without `noqa`ing) PEP8 violations and adjust my scipy-PR-reviewing criteria to be more flexible on this stuff.

Reply:
I pushed this proposed change to not hold this up any longer. I do think the old or new code are no worse or better, it's a matter of preference - so next time I'd prefer to just ignore, iterating on this kind of thing just adds friction imho.

The `noqa` is by far the worst option I believe, I dislike it almost as much as `+DOCTEST_SKIP`. Adding 12 characters of weird markup because a line is 1 character too long is never a good tradeoff, unless you're really forced to do it for other reasons (which we're not here, it won't cause CI failures).